src/App.js

```diff
@@ -1,27 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Please don't disable eslint rules. This is one of the basic requirements.

Reply:
you don't need to disable this, as you've used the variable so ESLint won't give you an error